data/tiles.yml

```diff
@@ -1,30 +1,332 @@
 tiles:
-  - planets:
+  - name: AbyzFria
+    planets:
       - name: Abyz
+        resources: 3
         influence: 0
-        resource: 3
+        trait: Hazardous
       - name: Fria
+        resources: 2
+        influence: 0
+        trait: Hazardous
```

Comment:
I was thinking each tile was have 'type' designation that would help determine things like how it should be rendered and to help with legal placement logic

Reply:
I thought of that too, but wasn't sure what to call "basic" tiles.  We have "Home" and "Anomaly" though.